Client/Coordinators/LibraryCoordinator.swift

```diff
@@ -35,24 +35,14 @@ class LibraryCoordinator: BaseCoordinator, LibraryPanelDelegate {
         libraryViewController.resetHistoryPanelPagination()
     }
 
-    func libraryPanelDidRequestToSignIn() {
-        // TODO: Will be handled by FXIOS-6604
-    }
-
-    func libraryPanelDidRequestToCreateAccount() {
-        // TODO: Will be handled by FXIOS-6604
-    }
+    // MARK: - LibraryCoordinatorDelegate
 
-    func libraryPanel(didSelectURL url: URL, visitType: VisitType) {
-        // TODO: Will be handled by FXIOS-6604
-    }
-
-    func libraryPanel(didSelectURLString url: String, visitType: VisitType) {
-        // TODO: Will be handled by FXIOS-6604
+    func libraryPanelDidRequestToOpenInNewTab(_ url: URL, isPrivate: Bool) {
+        parentCoordinator?.libraryPanelDidRequestToOpenInNewTab(url, isPrivate: isPrivate)
     }
 
-    func libraryPanelDidRequestToOpenInNewTab(_ url: URL, isPrivate: Bool) {
-        // TODO: Will be handled by FXIOS-6604
+    func libraryPanel(didSelectURL url: URL, visitType: Storage.VisitType) {
+        parentCoordinator?.libraryPanel(didSelectURL: url, visitType: visitType)
```

Comment:
We can have tests for those to ensure the parent coordinator delegate is called as expected